gcc/testsuite/rust/compile/issue-3307.rs

```diff
@@ -0,0 +1,9 @@
+#[lang = "sized"]
+trait Sized {}
```

Comment:
```suggestion
// { dg-additional-options "-frust-name-resolution-2.0" }

#[lang = "sized"]
trait Sized {}
```

Reply:
I added this patch. Wouldn't this cause issue-3304.rs to be run with NR2.0 twice?

Reply:
Yes, but it shouldn't matter -- that line should be alright either present or not present

Reply:
I agree, I'm happy to keep it there since it clarifies that this is specifically a NR2.0 test